pyscripts/test_mask.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
+
+import unittest
+import os
+import yaml
+import base64
+from mask import SecretMask
+
+# sample directory used to load yaml files
+SAMPLE_DIR = "tests/samples"
```

Comment:
@Sandeepyadav93 added `unittests` here: it reads sample files in the `tests/samples` directory and applies the `mask()` function. At the end we assert that the content of the `expected` file (the result of the processing) is not the same of the original one (or it's the same in case of `nochange.yaml`).

Reply:
thanks for info!